shared/desktop/yarn-helper/build.tsx

```diff
@@ -8,13 +8,12 @@ const outputStats = false
 
 const commands = {
   'build-dev': {
-    env: {BABEL_ENV: 'yarn', NO_SERVER: 'true'},
```

Comment:
react refresh has guards in place to complain if you run in babel env !==`'development'` . we were using `yarn` in our scripts since our node env and electron is slightly different but since electron is quite updated we're not using anything thats really different and can likely just ignore this concept now